rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.adoc

```diff
@@ -37,6 +37,7 @@ include::modules/osd-aws-privatelink-firewall-prerequisites.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
 == Additional resources
-* xref:../../rosa_planning/rosa-limits-scalability.adoc#rosa-limits-scalability[Limits and scalability]
+// Removed as part of OSDOCS-13310, until figures are verified.
```

Comment:
I cannot find this assembly under Install ROSA Classic Clusters. Can you provide a direct link?

Reply:
That one is pretty buried, under non-STS clusters in the Classic install doc: https://88263--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs

Reply:
Thanks. Given that this link was removed, the following paragraph should also be removed (EC2 instances section):

![Screenshot From 2025-03-19 09-00-40](https://github.com/user-attachments/assets/6ac2c3fe-b041-4b70-9a95-55d76387195c)

Reply:
Added a condition to remove this in non-HCP builds